### ISSUE TITLE
Code Contribution: [Hard] [Operator Development] mse_loss

### ISSUE THREAD

**Issue by StrongSpoon** [FlagOpen/FlagGems#396]
## Description 任务介绍
Develop forward function for mse_loss operator.
开发mse_loss算子的前向功能。
## Requirements 任务要求
**Interface 接口**
`mse_loss(Tensor self, Tensor target, int reduction=Mean) -> Tensor`
**Function reference 功能参考**
https://pytorch.org/docs/stable/generated/torch.nn.functional.mse_loss.html
**Implementation reference 实现参考**
None暂无

The operator should support all optional arguments defined in the interface. 
算子应支持接口中定义的所有参数选项。
Please provide both accuracy test and performance test code.
请同时提供实现正确性测试与性能测试代码。
## DDL 提交时间
Please submit a Pull Request within 3 weeks after accepting the assignment.
请于接取任务后三周内提交PR。

**Comment by awayzjj** [comment]
Binary 认领
